src/components/MapImageSlider.vue

```diff
@@ -1869,7 +1868,7 @@
       
           if (!this.firstHover) {
             document.getElementById("annotate-svg").setAttribute("class","hidden");
-            document.getElementById("arrow").setAttribute("class","hidden");
+            document.getElementById("annotate-arrow").setAttribute("class","hidden");
             this.firstHover = true;
```

Comment:
I think potentially this code is what was conflicting with the intro arrow because I had given it `id = "arrow"` around the same time it disappeared. I've removed that from the infographic code so hopefully we are good on that now